modules/007-registrypackages/images/kubeadm-centos/scripts/install

```diff
@@ -14,14 +14,30 @@
 # limitations under the License.
 
 set -Eeo pipefail
-  PACKAGES_TO_INSTALL=""
-if ! rpm --quiet -q "$(rpm -qip cri-tools.x86_64.rpm | grep Name | awk '{print $3}')"; then
-  PACKAGES_TO_INSTALL="cri-tools.x86_64.rpm"
+packages_to_install=""
+package_cri_tools="cri-tools"
+package_file_cri_tools="${package}.x86_64.rpm"
```

Comment:
i am not looking for `package` variable, maybe here should be `package_file_cri_tools`?

Reply:
yes it is error, thanks